src/js/components/ListItem.js

```diff
@@ -47,5 +47,6 @@ ListItem.defaultProps = {
   align: 'center',
   direction: 'row',
   pad: {horizontal: 'medium', vertical: 'small'},
-  separator: 'bottom'
+  separator: 'bottom',
+  role: 'listitem'
```

Comment:
I like the idea of having a default prop for role. but we still need the transfer props. 

I'm ok with leaving role prop if we decide to keep default props.